server.go

```diff
@@ -64,12 +64,17 @@ func handleComments(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		// Write out the json response
-		commentsEncoder := json.NewEncoder(cFile)
-		commentsEncoder.Encode(comments)
+		// Pretty print the JSON, write to file
+		var encodedComments, _ = json.MarshalIndent(comments, "", "    ")
```

Comment:
3 problems here:

1. The use of `var encodedComment...` vs `encodedComments :=`
2. ignoring the error is bad.
3. This causes the entirety of the encoded data to be stored in ram in encodedComments. The previous solution handled it via streaming readers and writers.